process/project_proposals.adoc

```diff
@@ -70,8 +70,13 @@ image::incubation-process.png[Incubation process]
 
 === Graduation process
 
-Graduation process follows the pattern of incubation, but the majority of DD should have happened at the incubation phase so it should be a less onerous process for graduation.
-
+. *Submit Graduation Proposal Template*
+   * Project fills out and submits the link:graduation-proposal-template.md[graduation proposal template] in a pull request in the https://github.com/cncf/toc[cncf/toc GitHub repo].
+   * The file containing the proposal should be located in https://github.com/cncf/toc/proposals/graduation/[the graduation proposals directory].
+   * The proposal addresses how the project has grown since incubation and any concerns from incubation DD in addition to the standard graduation requirements.
+. *Kick off two week period of time for pulic comment on the TOC mailing list*
+   * Project maintainers can send email on TOC mailing list to initiate this public comment period which should contain a link to the proposal pull request.
+   * All SIGs, end users, TOC members, and community members are welcome to comment at this time on the mailing list.
 . *TOC vote*
    * TOC members assess whether project meets the https://github.com/cncf/toc/blob/master/process/graduation_criteria.adoc#graduation-stage[Graduation criteria]
    * Projects get accepted to incubation via a 2/3 supermajority vote of the TOC
```

Comment:
Shouldn't this be:
`Projects get graduated via a 2/3 supermajority vote of the TOC`?

Reply:
Yes thank you. Updated.